drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/editor/ScenarioSimulationEditorPresenter.java

```diff
@@ -323,19 +246,25 @@ public void setRedoButtonEnabledStatus(boolean enabled) {
         view.getRedoMenuItem().setEnabled(enabled);
     }
 
-    @Override
-    public void addDownloadMenuItem(FileMenuBuilder fileMenuBuilder) {
-        fileMenuBuilder.addNewTopLevelMenu(view.getDownloadMenuItem(getPathSupplier()));
+    public void addDownloadMenuItem(FileMenuBuilder fileMenuBuilder, Supplier<Path> pathSupplier) {
+        fileMenuBuilder.addNewTopLevelMenu(view.getDownloadMenuItem(pathSupplier));
     }
 
     public DataManagementStrategy getDataManagementStrategy() {
         return dataManagementStrategy;
     }
 
     public void onImport(String fileContents) {
-        importExportService.call(getImportCallBack(),
-                                 getImportErrorCallback())
-                .importSimulation(CSV, fileContents, context.getStatus().getSimulation());
+        GWT.log(this.toString() + "onImport(" + fileContents + ")");
```

Comment:
Is this log necessary?

Reply:
@yesamer 
All the logs are leftover to be removed.

Reply:
Done